src/actions/apply_attack_action.rs

```diff
@@ -211,6 +269,79 @@ fn forecast_effect_attack(
         AttackId::A2a071ArceusExUltimateForce => {
             bench_count_attack(acting_player, state, 70, 20, None)
         }
+        // A2 Probabilistic Damage Attacks - Single Coin
+        AttackId::A2016WormadamLeafCutter => {
+            probabilistic_damage_attack(vec![0.5, 0.5], vec![0, 30])
```

Comment:
Note, this is missing the base damage of the attack. Its 60 on tails, and 90 on heads, since its 30 _additional_ damage. Double-check the rest of the attacks for this error (i.e. please consider the base damage accordingly).